cerbero/packages/wix.py

```diff
@@ -71,21 +71,20 @@ def __init__(self, config, package):
         # wix_wine_prefix is a symbolic link to the regular cerbero prefix
         # and should not be a folder.
         if self._with_wine:
-          self.wix_wine_prefix = '/tmp/flu_cwwp'
-          if os.path.exists(self.wix_wine_prefix):
-            os.remove(self.wix_wine_prefix)
-          os.symlink(config.prefix, self.wix_wine_prefix)
-          self.prefix = self.wix_wine_prefix
+          if os.path.exists(package.wix_wine_prefix):
```

Comment:
wix_wine_prefix shouldn't be a package variable.
Creating the symlink at this level is not the correct approach, it should be created at the most top level of the packaging step, in MSIPackager, so you ensure that it's created only once at the beginning of the packaging process.
Note that there is alreay a similar approach taken when a package needs stripping, that creates temporary directory and sets the prefix variable in the merge module.
See  https://github.com/fluendo/cerbero/blob/b11acf9bb9abc3b95948ce962ff0c3b2f26f6c6f/cerbero/packages/wix_packager.py#L68
and
https://github.com/fluendo/cerbero/blob/b11acf9bb9abc3b95948ce962ff0c3b2f26f6c6f/cerbero/packages/wix_packager.py#L91